db/migrate/20181012080059_create_article_categories.rb

```diff
@@ -0,0 +1,8 @@
+class CreateArticleCategories < ActiveRecord::Migration
+  def change
+    create_table :article_categories do |t|
+      t.integer :article_id
```

Comment:
It's better to use `add_reference` in here, because it will make sure MySQL also have reference integrity, relying on Rails Application Record association is not enough
See https://api.rubyonrails.org/v5.2.1/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference